Library/Homebrew/cmd/style.rb

```diff
@@ -111,12 +111,21 @@ def check_style_impl(files, output_type, options = {})
       args << "--only" << cops_to_include.join(",")
     end
 
+    has_non_formula = files.nil? || files.any? do |file|
```

Comment:
`Array(files).any?` to avoid the `nil?` check.

Reply:
Or `has_formula = files.none? { … }`.